### ISSUE TITLE
[Bug]: (Unknown Error - Code: X-5 occurred) + JSON TypeError

### ISSUE THREAD

**Issue by Sparkkkkie** [ThioJoe/YT-Spammer-Purge#563]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Well, I was trying to scan my comments on my channel but then this error came, i dont know what i can do about it but it does say its probs your fault, it also says your path and name of your pc, i know this has been reported already, but the X-5 is not reported yet.

### Release version

2.15.0-beta2

### Steps to reproduce

1. download beta-2 (i think)
2. connect yt channel.
3. execute it and use smart auto 
4. boom error

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Choice (1-7): 1

----------------------------------------------- Auto-Smart Mode -----------------------------------------------
~~~ This mode is a spammer's worst nightmare. It automatically scans for multiple spammer techniques ~~~

 > Extremely low (near 0%) false positives
 > Detects whatsapp scammers and '18+ spam' bots
 > Easily cuts through look-alike characters and obfuscations, including impersonating usernames
 > NOTE: This mode prioritizes a VERY low false positive rate, at the cost of occasionally missing some spammers.



------------------------------------------------------------------------------
(Note: If the program appears to freeze, try right clicking within the window)

                          --- Scanning ---

Traceback (most recent call last):canned: 54 | Matches Found So Far: 0
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1591, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1568, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1124, in primaryInstance
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1101, in scan_video
  File "Scripts\operations.py", line 142, in get_comments
TypeError: 'NoneType' object is not iterable
------------------------------------------------
Unknown Error - Code: X-5 occurred. This is probably my fault,
please a bug report on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report

 Press Enter to Exit...
```

### Screenshots

![Screenshot 2022-02-05 133911](https://user-images.githubusercontent.com/98154338/152642543-6bfccb27-8070-463b-9a83-499c76e54255.png)

**Comment by Kurdakov** [comment]
see here https://github.com/ThioJoe/YT-Spammer-Purge/issues/544

**Comment by Sparkkkkie** [comment]
ok, well im using beta 2 not 1

**Comment by Kurdakov** [comment]
I mean in case you are ok to switch to running python code
 and could pull beta branch https://github.com/ThioJoe/YT-Spammer-Purge/tree/2.15.0-Beta you might test

**Comment by Firecul** [comment]
> I mean in case you are ok to switch to running python code and could pull beta branch https://github.com/ThioJoe/YT-Spammer-Purge/tree/2.15.0-Beta you might test

I just crashed there so I'd just stick with the releases. I'll try to narrow down the problem and create an issue myself.

**Comment by Sparkkkkie** [comment]
I guess its just cause im using the beta, first time i get an error with a beta release of something lol

**Comment by ThioJoe** [comment]
This is a dupe of #544, it's already fixed for the next release. Though @Firecul seems to have found another bug which I'll have to fix so I'll keep this open

**Comment by KendallDoesCoding** [comment]
> This is a dupe of #544, it's already fixed for the next release. Though @Firecul seems to have found another bug which I'll have to fix so I'll keep this open

What about #558 ?

**Comment by Sparkkkkie** [comment]
> 

wow you actually replied lol, anyways thanks for telling me!

**Comment by Sparkkkkie** [comment]
> 

thats X-2 error

**Comment by ThioJoe** [comment]
Ok new release is out with the fix, 2.15.0-Beta3
Commit 4c3943296d6d2a96a365b5650d8bfbfcdafaeef3

**Comment by Sparkkkkie** [comment]
> Ok new release is out with the fix, 2.15.0-Beta3
> Commit [4c39432](https://github.com/ThioJoe/YT-Spammer-Purge/commit/4c3943296d6d2a96a365b5650d8bfbfcdafaeef3)

Alr ill test it when im back kn my laptop again
